cockroachcloud/pci-dss.md

```diff
@@ -0,0 +1,105 @@
+      </ol>
+  </td>
+</tr>
+<tr>
+  <td><i>Implement Strong Access Control Measures</i></td>
+  <td><ol start="7">
+        <li>Restrict access to cardholder data by business need to know</li>
+        <li>Identify and authenticate access to system components</li>
+        <li>Restrict physical access to cardholder data</li>
+      </ol>
+  </td>
+</tr>
+<tr>
+  <td><i>Regularly Monitor and Test Networks</i></td>
+  <td><ol start="10">
+        <li>Track and monitor all access to network resources and cardholder data.</li>
+        <li>Regularly test security systems and processes</li>
+      </ol>
+  </td>
+</tr>
+<tr>
+  <td><i>Maintain an Information Security Policy</i></td>
+  <td><ol start="12">
+        <li>Maintain a policy that addresses information security for all personne.</li>
+  </td>
+</tr>
+</tbody>
+</tgroup>
+</table>
+
+{{ site.data.products.dedicated }} is certified by a PCI Qualified Security Assessor (QSA) to be compliant with PCI DSS, as long as you take the actions outlined in [How can you ensure your cluster's compliance with PCI DSS?](#how-can-you-ensure-your-clusters-compliance-with-pci-dss).
+
+## How can you ensure your cluster's compliance with PCI DSS?
+
+Compliance is a shared responsibility. Although {{ site.data.products.dedicated }} clusters automatically comply with PCI DSS, you must take additional steps to ensure that your entire solution is compliant with PCI DSS, such as implementing security recommendations and carefully choosing business partners and vendors. For example, you must ensure that cardholder data is protected throughout its journey into and out of your {{ site.data.products.dedicated }} clusters.
+
+Cockroach Labs cannot provide specific advice about what is required for compliance with PCI DSS or how to implement a specific requirement. The following points help to illustrate some steps that organizations might take to help to ensure compliance..
+
+{{site.data.alerts.callout_danger}}
+The following list is provided only to illustrate a small subset of the measures you might need to take to ensure compliance.
+{{site.data.alerts.end}}
+
+- Before you insert cardholder data into the cluster, you must first protect it by a combination of encryption, hashing, masking, and truncation. For an example implementation, refer to [Integrate CockroachDB Dedicated with Satori](/docs/{{site.versions["stable"]}}/satori-integration.html).
+- The cryptographic materials used to protect cardholder data must themselves be protected at rest and in transit, and access to the unencrypted key materials must be strictly limited only to approved individuals.
+- Within the cluster, you must restrict access to cardholder data on a "need to know basis" authorized individuals have access to tables and views in the cluster that contain cardholder data, and you must regularly test for compliance. Refer to [Authorization](/docs/{{site.versions["stable"]}}/authorization.html).
```

Comment:
This is hard to parse: you must restrict access to cardholder data on a "need to know basis" authorized individuals have access to tables and views in the cluster that contain cardholder data

Reply:
I reworded this.